static/scripts/app-login.js

```diff
@@ -33,46 +38,41 @@ formbtn.addEventListener('click', async (e) => {
           },
         body: jsonData
     });
-    switch (datafetch.status) {
-        case 200:
-            console.log('successful login');
-            location.reload();
-            return;
-        case 302:
-        case 303:
-            console.log('repetitive login attempt was redirected');
-            location.reload();
-            return;
-        case 400:
-            console.log(datafetch.statusText, 'Empty values.')
-            return
-        case 401:
-            const inputsContainer = document.querySelector('.inputs-container');
-            let user_err = document.createElement('small');
-                    user_err.classList.add('big-user-err');
-                    user_err.innerText = `Invalid username or password.`;
-                    // remove err if already exist
-                    if(inputsContainer.firstChild.classList?.contains('big-user-err')){
-                        inputsContainer.firstChild.remove();
-                    }
-                    inputsContainer.prepend(user_err);
-
-                    setTimeout(() => {
-                        user_err.remove();
-                    }, 5000);
-            console.log(datafetch.statusText, 'Invalid username or password.')
-            return
-        case 500:
-            console.log(datafetch.statusText, '')
-            return
-        default:
-            console.log(`unexpected status received: ${datafetch.status}`)
-    }
-    let result = await datafetch.json();
-    console.log(result)
-    console.log(jsonData)
 
+    const result = await datafetch.json();
     
+    if(datafetch.ok){
+        let user_succ = document.createElement('small');
+        user_succ.classList.add('big-user-succ');
+        user_succ.innerText = `Success!`;
+        // remove err if already exist
+        if(inputsContainer.firstChild.classList?.contains('big-user-succ')
+         || inputsContainer.firstChild.classList?.contains('big-user-err')){
+            inputsContainer.firstChild.remove();
+        }
+        inputsContainer.prepend(user_succ);
+        setTimeout(() => {
+            user_succ.remove();
+        }, 5000);
+        console.log('successful login');
+        location.reload();
+        return;
+    }
+        let user_err = document.createElement('small');
+        user_err.classList.add('big-user-err');
+        user_err.innerText = `${result.message}`;
+        // remove err if already exist
+        if(inputsContainer.firstChild.classList?.contains('big-user-err')){
+            inputsContainer.firstChild.remove();
+        }
+        inputsContainer.prepend(user_err);
+        setTimeout(() => {
+            user_err.remove();
+        }, 5000);
```

Comment:
it these setTimeout calls, you are using a constant of 5 seconds, right? Have you considered putting this to some config or a const.js file so that if we need to change it, there is exactly 1 place to do so?

Reply:
yep, i can add this to template-script.js

Reply:
Create an asana, will do this week